transform.ml

```diff
@@ -1,7 +1,7 @@
 open Instr
 
 (* types of transformations *)
-type transform_instructions = instruction_stream -> instruction_stream option
+type transform_instructions = analysis_input -> analysis_input option
```

Comment:
no, but the reason was that it allows one to combine transformations with &&&.
but now that I think about it, the &&& operator could take care of pushing it through!